.tugboat/steps/1-init.sh

```diff
@@ -72,24 +66,26 @@ nodejs -v | grep -q v$NODE_MAJOR
 #drainpipe-start
 # This is necessary for testing as this repository doesn't hold a Drupal site.
 shopt -s dotglob
-mkdir ../drainpipe-tmp
-mv * ../drainpipe-tmp/
-composer create-project drupal/recommended-project .
-mv ../drainpipe-tmp drainpipe
+rm -rf /var/www/html/*
+composer create-project drupal/recommended-project /var/www/html
+ln -snf "/var/www/html/web" "${DOCROOT}"
+cd /var/www/html
 composer config extra.drupal-scaffold.gitignore true
 composer config --json extra.drupal-scaffold.allowed-packages \[\"lullabot/drainpipe\"]
 composer config --no-plugins allow-plugins.composer/installers true
 composer config --no-plugins allow-plugins.drupal/core-composer-scaffold true
 composer config --no-plugins allow-plugins.lullabot/drainpipe true
-composer config repositories.drainpipe --json '{"type": "path", "url": "drainpipe", "options": {"symlink": true}}'
+composer config repositories.drainpipe --json "{\"type\": \"path\", \"url\": \"${TUGBOAT_ROOT}\", \"options\": {\"symlink\": true}}"
 composer config extra.drainpipe --json '{"tugboat": {}}'
 composer config minimum-stability dev
 composer require lullabot/drainpipe --with-all-dependencies
 cp web/sites/default/default.settings.php web/sites/default/settings.php
 #drainpipe-end
 
+# Create the Drupal private and public files directories if they aren't
+# already present.
+mkdir -p "${DOCROOT}/sites/default/files"
+chmod 777 "${DOCROOT}/sites/default/files"
+chgrp -R www-data "${DOCROOT}/sites/default/files"
+
 composer install
-#drainpipe-start
-rm -rf .tugboat
```

Comment:
This logic is not needed anymore, because Drainpipe is installed in `/var/www/html` and the directory `${TUGBOAT_ROOT}/.tugboat` remains unaltered.